builder/builder.js

```diff
@@ -37,41 +38,48 @@ module.exports = (options) => {
     cssDir,
     environment = 'development',
     jsDir,
-    output = defaultOutputThemeFile,
-    themeFile,
     watch,
   } = options;
 
-  const themeFiles = [defaultThemeYamlPath].concat(themeFile).filter(Boolean);
+  const bundles = [{
+    themeFiles: [defaultThemeYamlPath],
+    output: path.join(defaultOutputThemeDir, 'default'),
+  }];
+  getAvailableSets().forEach((themeOptions) => {
+    bundles.push({
+      themeFiles: [defaultThemeYamlPath].concat(getThemeFiles(themeOptions)),
```

Comment:
I would recommend simplifying a bit here preparing data before calling the `push`. It will increase readability.
```
getAvailableSets().forEach((themeOptions) => {
  const themeFiles = [defaultThemeYamlPath].concat(getThemeFiles(themeOptions));
  const output = path.join(defaultOutputThemeDir, `${themeOptions.brand}-${themeOptions.affiliate}`);
  bundles.push({themeFiles, output});
});
```